Makefile

```diff
@@ -70,9 +71,17 @@ fix-lint:
 	golangci-lint run --fix
 
 .PHONY: verify-shellcheck
-verify-shellcheck:
+verify-shellcheck: install-shellcheck
 	PASSES="shellcheck" ./scripts/test.sh
 
+.PHONY: install-shellcheck
+install-shellcheck:
+	if ! command -v shellcheck > /dev/null 2>&1; then \
+        wget -qO- "https://github.com/koalaman/shellcheck/releases/download/${SHELLCHECK_VERSION}/shellcheck-${SHELLCHECK_VERSION}.linux.x86_64.tar.xz" | tar -xJv; \
```

Comment:
Just out of curious, do we need to conside other os distro and arch? like darwin and x86?

Reply:
Currently, etcd dev supports only **linux-amd64**. Check [Setup development environment][1].

[1]: https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#setup-development-environment